.gitignore

```diff
@@ -90,3 +90,4 @@ nb-configuration.xml
 ## Logs
 ##############################
 **/*.log
+/src/main/resources/stripe.yaml
```

Comment:
Are we sure we don't need this file?

Reply:
> Are we sure we don't need this file?

Stripe.yaml file with sensitive data. Such file cannot be shared in github